portal-api/src/v1/api.js

```diff
@@ -287,7 +299,12 @@ const getUtilisationReports = async (bankId, reportPeriod) => {
       console.error('Get utilisation reports failed with the following report period: %s', reportPeriod);
       throw new Error('Invalid report period provided: %s', reportPeriod);
     }
-    const params = reportPeriod ? reportPeriod.start : undefined;
+
+    if (reportStatuses?.length > 0 && !areValidReportStatuses(reportStatuses)) {
```

Comment:
I think `areValidReportStatuses` is a bit unnecessary of a function. I'd probably go with the individual `isValidStatus` function and then do `reportStatuses.every(isValidStatus)`